js/src/logger.ts

```diff
@@ -736,7 +914,16 @@ function logFeedbackImpl(
         [IS_MERGE_FIELD]: true,
       };
     });
+    const recordAttachmentUpdate = new LazyValue(async () => {
+      return {
+        id,
+        ...JSON.parse(JSON.stringify(filteredUpdateEvent)),
```

Comment:
I wonder if we should roundtrip the `updateEvent` while we're here, so we don't forget about it.

Reply:
Done, deep copying before all log callsites